content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+on cloud security since every cloud provider and workload is different. We will
+however provide links to some of the popular cloud providers' own documentation
+for security as well as give general guidance for securing the infrastructure that
+makes up a Kubernetes cluster.
+
+### Cloud Provider Security Table
+
+
+
+IaaS Provider        | Link |
+-------------------- | ------------ |
+Amazon Web Services | https://aws.amazon.com/security/ |
+Google Cloud Platform | https://cloud.google.com/security/ |
+Microsoft Azure | https://docs.microsoft.com/en-us/azure/security/azure-security |
+IBM Cloud | https://www.ibm.com/cloud/security |
+Alibaba Cloud | https://www.alibabacloud.com/trust-center |
+
+If you are running on your own hardware or a different cloud provider you will need to
+consult your documentation for security best practices.
+
+### General Infrastructure Guidance Table
+
+Area of Concern for Kubernetes Infrastructure | Recommendation |
+--------------------------------------------- | ------------ |
+Network access to API Server (Masters) | Ideally all access to the Kubernetes Masters is not allowed publicly on the internet and is controlled by network access control lists restricted to the set of IP addresses needed to administer the cluster.|
+Network access to Nodes (Worker Servers) | Nodes should be configured to _only_ accept connections (via network access control lists) from the masters on the specified ports, and accept connections for services in Kubernetes of type `NodePort` and `LoadBalancer`. If possible, this nodes should not exposed on the public internet entirely.
+Kubernetes access to Cloud Provider API | Each cloud provider will need to grant a different set of permissions to the Kubernetes Masters and Nodes, so this recommendation will be more generic. It is best to provide the cluster with cloud provider access that follows the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) for the resources it needs to administer. An example for Kops in AWS can be found here: https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md#iam-roles
+Access to etcd | Access to etcd (the datastore of Kubernetes) should be limited to the masters only. Depending on your configuration you should also attempt to use etcd over TLS. More info can be found here: https://github.com/etcd-io/etcd/tree/master/Documentation#security
+etcd Encryption | Wherever possible it's a good practice to encrypt all drives at rest, but since etcd holds the state of the entire cluster (including Secrets) its disk should especially be encrypted at rest.
+
+## Cluster
+
+This section will provide links to more exploration for securing the aspects of
+workloads in Kubernetes. There are essentially two areas of concern for securing
+Kubernetes:
+
+* Securing the components that are configurable which _make up_ the cluster
+* Securing the components which run _in_ the cluster
+
```

Comment:
italics needed?

Reply:
I was trying to highlight the difference, deleting